api/definitions.yaml

```diff
@@ -211,11 +211,11 @@ IsInboxEnabled:
   default: false
   description: True if the recipient of a message wants to store its content for
     later retrieval.
-TimeToLive:
+TimeToLiveSeconds:
   type: integer
   default: 3600
   minimum: 3600
-  maximum: 31536000
+  maximum: 604800
```

Comment:
hehe told you :D even the most experienced programmers get caught by these kind of bugs, keeping track of unit of measure is hard, esp. re time